Security/Http/ResourceOwnerMap.php

```diff
@@ -65,6 +65,16 @@ public function setContainer(ContainerInterface $container = null)
     {
         $this->container = $container;
     }
+    
+    /**
+     * Gets service container
+     *
+     * @return container
+     */
+    public function getContainer()
```

Comment:
Certainly not. :-)

Please use dependency injection.

This is already quite strange to have the container here...